src/backend/api/Fusion.Resources.Domain/Extensions/PositionV2Extensions.cs

```diff
@@ -28,7 +28,7 @@ public static bool HasActiveAssignment(this ApiPositionV2? position, Guid azureU
             if (position.Instances == null)
                 return null;
 
-            return position.Instances.FirstOrDefault(i => i.AppliesFrom <= DateTime.Today && i.AppliesTo >= DateTime.Today);
+            return position.Instances.FirstOrDefault(i => i.AppliesFrom.Date <= DateTime.Today && i.AppliesTo.Date >= DateTime.Today);
         }
```

Comment:
Looks OK.
When using Date component you would get from 00:00:00, so I guess only the first modification here will make any difference.

Maybe you could consider verifying/validating extension HasActiveAssignment (in same file), regarding AppliesFrom :-)
Last but not least. There is also a similar extension in the API project called HasActiveAssignment.

Reply:
Good catch, will consolidate these. The one in the API can actually be removed.